src/routes/Apps/App/LogDialogs.tsx

```diff
@@ -523,19 +524,22 @@ class LogDialogs extends React.Component<Props, ComponentState> {
                     />
                 </div>
                 <OF.SearchBox
+                    data-testid="search-box"
```

Comment:
These ids should all be prefixed with the component object to guarantee uniqueness